etc/ReplayOfflineConfiguration.py

```diff
@@ -100,22 +100,22 @@
 
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_11_3_2"
+    'default': "CMSSW_12_0_0"
 }
 
 # Configure ScramArch
 setDefaultScramArch(tier0Config, "slc7_amd64_gcc900")
```

Comment:
The easiest way to handle the arch issue is  changing the default arch to the old `slc7_amd64_gcc630` and using `slc7_amd64_gcc900` only in the case of using  `CMSSW_12_0_0`.

```suggestion
setDefaultScramArch(tier0Config, "slc7_amd64_gcc630")
setScramArch(tier0Config, "CMSSW_12_0_0", "slc7_amd64_gcc900")
```